source/boards/getting-started.rst

```diff
@@ -0,0 +1,59 @@
+Getting Started
+===============
+
+Mattermost Focalboard is included in the Mattermost Cloud workspace, enabled by default, and upgraded automatically.
+
+For self-managed deployments, Focalboard is available in the Plugin Marketplace.
+
+1. As a System Admin, go to **Main Menu > Plugin Marketplace**.
+2. Search for **Focalboard**.
+3. Select **Install** if not yet installed, then select **Configure** to enable.
+4. From the plugin configuration page, set **Enable Plugin** to **true**.
+5. Select **Save** to enable the plugin.
+
+Once installed and configured, the Focalboard plugin requires websocket traffic to be passed by the proxy. Update your NGINX or Apache web proxy config following the steps below.
+
+Updating the NGINX web proxy config
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+After following the standard `Mattermost install steps <https://docs.mattermost.com/install/install-ubuntu-1804.html#configuring-nginx-as-a-proxy-for-mattermost-server>`_, edit ``/etc/nginx/sites-available/mattermost`` and add this section to it:
```

Comment:
Question: Does a single underline work for links like this?

Reply:
I feel a bit dumb for asking but could you elaborate on that?

Reply:
I could have been much clearer, sorry! I'm seeing a mix of external link syntax throughout the source code that includes a single underline or two underlines trailing the URL. For all external links, I'm wondering if two underlines is required syntax, and if including a single underline still results in a clickable, functional link.

Reply:
Ooh I see what you mean! That's really weird. Let me look into it.